src/Aspire.Dashboard/Model/ResourceViewModel.cs

```diff
@@ -55,3 +55,5 @@ public sealed class ResourceServiceSnapshot(string name, string? allocatedAddres
     public int? AllocatedPort { get; } = allocatedPort;
     public string AddressAndPort { get; } = $"{allocatedAddress}:{allocatedPort}";
 }
+
+public sealed record EndpointViewModel(string EndpointUrl, string ProxyUrl);
```

Comment:
@drewnoakes Guessing with your refactorings and naming changes that this shouldn't be called EndpointViewModel anymore?

Reply:
This is night time for Drew. We can merge this and update in a separate PR when Drew replies back.

Reply:
I'm midway through that refactoring now, building on top of my latest PR. Please take a look at that PR if you have time, to help keep that work moving.